dbt_subprojects/daily_spellbook/models/chainswap/chain_swap_schema.yml

```diff
@@ -0,0 +1,204 @@
+        - &amount_usd
+          name: amount_usd
+          description: "USD value of the trade at time of execution"
+        - &type
+          name: type
+          description: "Wether the trade is a buy or sell"
+        - &token_bought_amount
+          name: token_bought_amount
+          description: "Value of the token bought at time of execution in the original currency"
+        - &token_bought_symbol
+          name: token_bought_symbol
+          description: "Token symbol for token bought in the trade"
+        - &token_bought_address
+          name: token_bought_address
+          description: "Contract address of the token bought"
+        - &token_sold_amount
+          name: token_sold_amount
+          description: "Value of the token sold at time of execution in the original currency"
+        - &token_sold_symbol
+          name: token_sold_symbol
+          description: "Token symbol for token sold in the trade"
+        - &token_sold_address
+          name: token_sold_address
+          description: "Contract address of the token sold"
+        - &fee_usd
+          name: fee_usd
+          description: "USD value of the fee at time of execution"
+        - &fee_token_amount
+          name: fee_token_amount
+          description: "Value of the fee paid at time of execution in the original currency"
+        - &fee_token_symbol
+          name: fee_token_symbol
+          description: "Token symbol for fee token"
+        - &fee_token_address
+          name: fee_token_address
+          description: "Contract address of the fee token"
+        - &project
+          name: project
+          description: "Project name of the DEX"
+        - &version
+          name: version
+          description: "Version of the contract built and deployed by the DEX project"
+        - &token_pair
+          name: token_pair
+          description: "Token symbol pair for each token involved in the trade"
+        - &project_contract_address
+          name: project_contract_address
+          description: "Project contract address which executed the trade on the blockchain"
+        - &user
+          name: user
+          description: "Address which initiated the trade"
+        - &tx_hash
+          name: tx_hash
+          description: "Unique transaction hash value tied to each transaction on the DEX"
+        - &evt_index
+          name: evt_index
+          description: "Index of the corresponding trade event"
+        - &is_last_trade_in_transaction
+          name: is_last_trade_in_transaction
+          description: "Wether the trade is the last hop of the trade transaction, in case of a multi-hop trade"
+
+  - name: chain_swap_ethereum_trades
+    meta:
+      blockchain: ethereum
+      sector: project
+      project: chain_swap
+      contributors: whale_hunter
+    config:
+      tags: ["ethereum", "dex", "chain_swap", "trades"]
+    description: >
+      ChainSwap trades on Ethereum
+    tests:
```

Comment:
By the way do you want to add `columns` here?
You can use * syntax to keep the list simple.

Reply:
not sure i understand what you mean here, for uniqueness i only want the 3 columns `blockchain`, `tx_hash`, `evt_index`

Reply:
Something like
```
columns:
  - *blockchain
  - *block_time
  - *block_date
  - *block_month
  - *amount_usd
  - *type
  - *token_bought_amount
  - *token_bought_symbol
  - *token_bought_address
  - *token_sold_amount
  - *token_sold_symbol
  - *token_sold_address
  - *fee_usd
  - *fee_token_amount
  - *fee_token_symbol
  - *fee_token_address
  - *project
  - *version
  - *token_pair
  - *project_contract_address
  - *user
  - *tx_hash
  - *evt_index
  - *is_last_trade_in_transaction
```

Reply:
still don't understand it, would you mind editing it yourself? appreciate it!